hie-ghc-mod/Haskell/Ide/GhcModPlugin.hs

```diff
@@ -31,10 +31,10 @@ ghcmodDescriptor = PluginDescriptor
 \from modern IDEs in any editor."
   , pdCommands =
          buildCommand checkCmd (Proxy :: Proxy "check") "check a file for GHC warnings and errors"
-                       [".hs",".lhs"] (SCtxFile :& RNil) RNil
+                       [".hs",".lhs"] (SCtxFile :& RNil) RNil ChangeCurrent
 
       :& buildCommand lintCmd (Proxy :: Proxy "lint")  "Check files using `hlint'"
-                     [".hs",".lhs"] (SCtxFile :& RNil) RNil
+                     [".hs",".lhs"] (SCtxFile :& RNil) RNil ChangeAll
```

Comment:
If that is the case, would you mind if I changed the description to "Checks a file using `hlint'"?